assets/js/modules/adsense/util/status.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * Status utlities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const ACCOUNT_STATUS_NONE = 'none';
+export const ACCOUNT_STATUS_DISAPPROVED = 'disapproved';
+export const ACCOUNT_STATUS_GRAYLISTED = 'graylisted';
+export const ACCOUNT_STATUS_MULTIPLE = 'multiple';
+export const ACCOUNT_STATUS_NO_CLIENT = 'no-client';
+export const ACCOUNT_STATUS_PENDING = 'pending';
+export const ACCOUNT_STATUS_APPROVED = 'approved';
+
+// TODO: If AdSense API exposes more information, site status can be determined
```

Comment:
I think what this is getting at is that if "if the future the AdSense API exposes more information, we can more specifically determine account status", right?

Maybe this could be updated to something like:

```
// TODO: Expand the statuses provided here to be more specific, but only based on
// the AdSense API.
// If the AdSense API exposes more information in future versions,
// site status can be described more accurately. For now, we only
// have these two functions, as making inferences based on the
// limited data we get back from the AdSense API has proved problematic
// in the past.
```

Something like that sums it up, right? 🤷‍♂